package.json

```diff
@@ -66,7 +66,7 @@
     "babel-plugin-add-module-exports": "^0.2.1",
     "babel-plugin-rewire": "^1.0.0",
     "babel-plugin-transform-object-assign": "^6.22.0",
-    "babel-plugin-transform-runtime": "^6.7.5",
+    "babel-plugin-transform-runtime": "^6.23.0",
```

Comment:
Old version of this PR had async/await, thats why I upgraded the dependency. But actually it should be removed completely.